src/sdk/namespace_fs.js

```diff
@@ -1941,15 +1951,18 @@ class NamespaceFS {
     async get_object_tagging(params, object_sdk) {
         const tag_set = [];
         let file_path;
+        let file;
+        // In case of object as a folder. folder tag can access using exact object key `folder1/` or folder name `folder1`
+        params.key = params.key.replace(/\/$/, '');
```

Comment:
Could you please save it in a separate variable?
Usually, we don't manipulate the parameters we receive directly.